scripts/Install-VSCode.ps1

```diff
@@ -226,11 +228,16 @@ function Get-CodePlatformInformation {
     }
 
     switch ($BuildEdition) {
-        'Stable' {
+        'Stable-System' {
             $appName = "Visual Studio Code ($Bitness)"
             break
         }
 
+        'Stable-User' {
+            $appName = "Visual Studio Code ($($Architecture) - User)"
```

Comment:
Any reason you used `$Architecture` here instead of `$Bitness` like System?

Reply:
No, I did the same as the Insiders Edition - user install.